tests/tr1/tests/atomic/test.cpp

```diff
@@ -222,6 +222,72 @@ static void test_atomic_flag() { // test properties of atomic_flag
     CHECK_INT(value, false);
 }
 
+#if _HAS_CXX20
+// TEST C++20 SEMANTICS OF atomic_flag
+static void test_atomic_flag_cxx20() { // test properties of atomic_flag
+    STD atomic_flag flag = ATOMIC_FLAG_INIT;

```

Comment:
`ATOMIC_FLAG_INIT` is deprecated / extraneous in C++20; the default constructor now initializes `atomic_flag` to the clear state.
```suggestion
    STD atomic_flag flag;
```